docs/install_guides/installing-openvino-pip.md

```diff
@@ -1,107 +1,142 @@
 # Install Intel® Distribution of OpenVINO™ Toolkit from PyPI Repository {#openvino_docs_install_guides_installing_openvino_pip}
 
+@sphinxdirective
+
 You can install both OpenVINO™ Runtime and OpenVINO Development Tools through the PyPI repository. This page provides the main steps for installing OpenVINO Runtime.
 
-> **NOTE**: From the 2022.1 release, the OpenVINO™ Development Tools can only be installed via PyPI. See [Install OpenVINO Development Tools](installing-model-dev-tools.md) for detailed steps.
+.. note:
+
+   From the 2022.1 release, the OpenVINO™ Development Tools can only be installed via PyPI. See :doc:`Install OpenVINO Development Tools <openvino_docs_install_guides_install_dev_tools>` for detailed steps.
 
-## Installing OpenVINO Runtime
 
-For system requirements and troubleshooting, see <https://pypi.org/project/openvino/>.
+Installing OpenVINO Runtime
+###########################
 
-### Step 1. Set Up Python Virtual Environment
+For system requirements and troubleshooting, see https://pypi.org/project/openvino/
 
-Use a virtual environment to avoid dependency conflicts. 
+Step 1. Set Up Python Virtual Environment
++++++++++++++++++++++++++++++++++++++++++
+
+Use a virtual environment to avoid dependency conflicts.
 
 To create a virtual environment, use the following command:
 
-@sphinxdirective
 
-.. tab:: Linux and macOS
+.. tab-set::
 
-   .. code-block:: sh
-   
-      python3 -m venv openvino_env
-   
-.. tab:: Windows
+    .. tab-item:: Linux and macOS
+        :sync: linmac
 
-   .. code-block:: sh
-   
-      python -m venv openvino_env
-     
-     
-@endsphinxdirective
+        .. code-block:: sh
 
-### Step 2. Activate Virtual Environment
+           python3 -m venv openvino_env
 
-@sphinxdirective
+    .. tab-item:: Windows
+        :sync: win
+
+        .. code-block:: sh
+
+           python -m venv openvino_env
 
-.. tab:: Linux and macOS
 
-   .. code-block:: sh
-   
-      source openvino_env/bin/activate
-   
-.. tab:: Windows
+Step 2. Activate Virtual Environment
+++++++++++++++++++++++++++++++++++++
+
+
+.. tab-set::
+
+    .. tab-item:: Linux and macOS
+        :sync: linmac
+
+        .. code-block:: sh
+
+           source openvino_env/bin/activate
+
+    .. tab-item:: Windows
+        :sync: win
+
+        .. code-block:: sh
+
+           openvino_env\Scripts\activate
 
-   .. code-block:: sh
-   
-      openvino_env\Scripts\activate
 
 .. important::
 
    The above command must be re-run every time a new command terminal window is opened.
 
-@endsphinxdirective
 
-### Step 3. Set Up and Update PIP to the Highest Version
+Step 3. Set Up and Update PIP to the Highest Version
+++++++++++++++++++++++++++++++++++++++++++++++++++++
 
 Use the following command:
-```sh
-python -m pip install --upgrade pip
-```
 
-### Step 4. Install the Package
+.. code-block: sh
+
+   python -m pip install --upgrade pip
```

Comment:
it's not rendered well.
at least in Safari I don't see this command
<img width="783" alt="Screenshot 2023-03-03 at 7 38 28 PM" src="https://user-images.githubusercontent.com/2566854/222762368-3d0cd6ea-fec5-4dfc-9f70-6c6b9287b62b.png">

Reply:
the issue is still here

Reply:
The issue also reproduces on Windows 11 Enterprise for the Chrome Version 110.0.5481.178 and Version 111.0.5563.65

Reply:
It's been fixed :)